Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,70 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+/*
+function duration(item) {
+  return item.duration;
+}
+function sum(prev, next) {
+  return prev + next;
+}
+console.log(tasks.map(duration).reduce(sum));
+*/
+
+/*
+let hours = 0;
+for (let i = 0; i < tasks.length; i++) {
+  const num = tasks[i].duration;
+  hours += num;
+}
+hours /= 60;
+console.log(hours);
+*/
+
+let hours = 0;
```

Comment:
We don't need this `hours` variable and the computations that are related to it.